### DISCUSSION TITLE
Dịch magic method trong Python

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#136] [Terminology] [answered]
Từ này nên dịch là:
* `phương thức đặc biệt`
* giữ nguyên là `phương thức magic`  nhỉ?

**Comment by rootonchair** [comment]
+1 giữ nguyên. Mấy từ này do Python tự đặt ra nên dịch ra em thấy không make sense lắm. Hoặc mình có thể giữ nguyên 1 lần để người đọc biết tên của các loại method này rồi về sau mình cứ dịch "phương thức đặc biệt"

**Reply by minhduc0711** [reply]
Nên để là "phương thức magic" hay nguyên "magic method" nhỉ :thinking:

**Reply by quangvnai** [reply]
Thế thì ta sẽ giữ nguyên là magic method, đóng mở ngoặc `phương thức đặc biệt` rồi sau đó thì dùng từ dịch nhỉ :3

**Reply by minhduc0711** [reply]
Không phải mình cũng làm vậy với các từ khóa khác à :)))

**Reply by quangvnai** [reply]
ĐIều đó có nghĩa là vote dịch từ `magic method` nhỉ :)))

**Comment by minhduc0711** [comment, marked as answer]
Mình chốt dịch "magic method" là "phương thức đặc biệt" nhé.
